stock_release_channel/models/stock_release_channel.py

```diff
@@ -202,13 +202,16 @@ class StockReleaseChannel(models.Model):
     )
     last_done_picking_name = fields.Char(compute="_compute_last_done_picking")
     last_done_picking_date_done = fields.Datetime(compute="_compute_last_done_picking")
+    collect_pickings = fields.Boolean(
+        compute="_compute_collect_pickings",
```

Comment:
Isn't there are `default=True` missing, or a migration script to not break the current behavior?
Since `_compute_collect_pickings` only setting `False`

Reply:
migration script added